LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) MapBox
```

Comment:
MapBox?

Reply:
We copied this from https://github.com/mapbox/variant/blob/master/LICENSE which still has `MapBox` on it. We also use the same license [here](https://github.com/mapbox/mapbox-base/blob/master/mapbox/io/LICENSE) and [here](https://github.com/mapbox/mapbox-base/blob/master/mapbox/weak/LICENSE).